### DISCUSSION TITLE
Use AI-on-the-edge for analog dial thermometer 0-120 degrees Celsius possible?

### DISCUSSION THREAD

**Discussion by Alzilander** [jomjol/AI-on-the-edge-device#3052] [General]
Dear Experts, 

is it possible to use the fantastic AI-on-the-Edge routine for a analog dial thermometer that shows between 0-120 degrees Celsius? I turned off the digital ROI and only use the analog ROI with one ROI (main). Unfortunately the identified values are wrong even I updated them several time manually. Any idea? Your help would be highly welcome :-) 

Thank you, 
Alex
![2024-04-28 12_49_31-Window](https://github.com/jomjol/AI-on-the-edge-device/assets/168340806/9c40a696-f399-4bb1-8a3a-e080e9c1d9ff)

**Comment by SybexX** [comment]
https://github.com/jomjol/AI-on-the-edge-device/discussions/3020#discussioncomment-9215783

![326251679-9c40a696-f399-4bb1-8a3a-e080e9c1d9ff](https://github.com/jomjol/AI-on-the-edge-device/assets/587201/6d97364c-4ebe-4493-932d-9b9ec8069f4c)
